crates/codegen/parser/runtime/src/templates/kinds.rs.jinja2

```diff
@@ -33,6 +33,34 @@ impl RuleKind {
     }
 }
 
+#[derive(
+    Debug,
+    Eq,
+    Ord,
+    PartialEq,
+    PartialOrd,
+    serde::Serialize,
+    strum_macros::AsRefStr,
+    strum_macros::Display,
+    strum_macros::EnumString,
+)]
+#[strum(serialize_all = "snake_case")]
+#[cfg_attr(feature = "slang_napi_interfaces", /* derives `Clone` and `Copy` */ napi(string_enum, namespace = "kinds"))]
+#[cfg_attr(not(feature = "slang_napi_interfaces"), derive(Clone, Copy))]
+pub enum FieldName {
+    // Built-in fields
```

Comment:
nit suggestion:

```suggestion
    // Built-in fields
    {% _SLANG_INTERNAL_RESERVED_NODE_FIELD_NAMES_ (keep in sync) %}
```

Reply:
Done in https://github.com/NomicFoundation/slang/pull/713/commits/062f6e6977ec62454f0ee849cdcfe649b7e45c8a